### DISCUSSION TITLE
synology-nas-hyper-backup-pause-solution/

### DISCUSSION THREAD

**Discussion by giscus** [fr0der1c/dev-blog-discussions#1] [Announcements]
# synology-nas-hyper-backup-pause-solution/

在群晖生态中，官方备份组件 Hyper Backup 大概是很多玩家做备份的第一选择。然而我实测发现备份过程中经常出现自动暂停的情况。在检查了错误发生的原因之后，我尝试使用浏览器自动化工具，进行暂停状态的监听和恢复。

https://dev.admirable.pro/synology-nas-hyper-backup-pause-solution/

**Comment by fr0der1c** [comment]
测试一下基于 giscus 的评论系统。
